pipeline/recon-pipeline.py

```diff
@@ -235,6 +235,16 @@ def do_scan(self, args):
         # luigi --module pipeline.recon.web.webanalyze WebanalyzeScan --target-file tesla --top-ports 1000 --interface eth0
         command = ["luigi", "--module", scans.get(args.scantype)[0]]
 
+        if args.target and args.target_file:
+            self.async_alert(style(f"[!] You can't use the --target and --target-file options in the same scan!"))
+            return
+        elif args.target:
+            with open("tmp_tgt", "w") as file:
+                file.write(args.target)
+                args.__statement__.arg_list[args.__statement__.arg_list.index("--target") + 1] = os.path.abspath(
```

Comment:
This project uses pathlib for dealing with file system paths.  For consistency, can you please swap `os.path.abspath` to its pathlib equivalent?  Additionally, add a `.expanduser()` call before `.resolve()` as both `.resolve` and `.abspath` can have surprising results.  There's an example [here](https://github.com/GreaterGoodest/recon-pipeline/blob/add-target/pipeline/recon-pipeline.py#L429)

```
>>> os.path.abspath("~/stuff")
'/tmp/recon-pipeline/~/stuff'
>>> Path("~/stuff").resolve()
PosixPath('/tmp/recon-pipeline/~/stuff')
```